resources/svc/default-postgresql.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.1'
 
 services:
-  db:
+  postgresdb:
```

Comment:
done (I have modified the postgres & mysql default docker yamls so this will actually work when we run the `start` command - previously it just keeps restarting when you run it)